lib/facility_factory.rb

```diff
@@ -0,0 +1,46 @@
+class FacilityFactory
+  def initialize
+  end
+
+  def create_facility(api)
+    api.map do |facility|
+      Facility.new({
+        name: format_name(facility),
+        address: format_address(facility),
+        phone: format_phone_number(facility) 
+      })
+    end
+  end
+
+  def format_name(facility)
+    if facility[:state] && facility[:state] == "MO"
+      facility[:name].rstrip
+    elsif facility[:state] && facility[:state] == "NY"
+      facility[:office_name]
+    else facility[:website] && facility[:website].include?('oregon')
```

Comment:
Typically we don't put a condition on the else. If we have another condition, it should be an `elsif` block. Then we _would_ want a default value in the else incase none of the above conditions are met.